wandb/sdk/artifacts/artifact_manifest_entry.py

```diff
@@ -68,7 +68,9 @@ def __repr__(self) -> str:
         size = f", size={self.size}" if self.size is not None else ""
         extra = f", extra={json.dumps(self.extra)}" if self.extra else ""
         local_path = f", local_path={self.local_path!r}" if self.local_path else ""
-        skip_cache = f", skip_cache={self.skip_cache!r}" if self.skip_cache else ""
+        skip_cache = (
+            f", skip_cache={self.skip_cache}" if self.skip_cache is not None else ""
+        )
```

Comment:
`skip_cache` can't be `None` now, right?

Reply:
oh yes, true. will update this